### DISCUSSION TITLE
Authentik as next SSO provider

### DISCUSSION THREAD

**Discussion by ojsef39** [tobychui/zoraxy#451] [Ideas]
Hey, i would love to see Authentik being added as the next SSO provider.

Love zoraxy, keep up the great work! :)

Also i would really love to see a implementation for the local user(s) as well, so you can log with your admin (role) user from SSO into Zoraxy 

And have a great new year 🎉

**Comment by morskipas** [comment]
Hello everyone,
I am also currently using Authentik in combination with nginx proxy manager. I have already tested zoraxy a bit and really like the additional features and of course the GUI. Authentik as an additional SSO provider would be outstanding.

Happy new year!!!

**Comment by tobychui** [comment]
For your information, as we currently do not have a dedicated maintainer for the SSO features, all the SSO features will only come from the community and I will only do the integration (UI and restructuring for easier future maintenance).  

**There are no timeline and plan for any additional SSO feature unless the module / core function has been implemented by the community contributor.** Thank you for your understanding and wish you all a happy new year!

**Reply by ojsef39** [reply]
I see, thank you very much for the clarification :)

Maybe when i find some time for it, ill look into it, but can't promise anything.

**Comment by crocofied** [comment]
Are there any new updates about this?

**Comment by JokerQyou** [comment]
I've created #568 to discuss this. It's somewhat functional now but has some issues. I also suspect Authentik struggles with external outposts when it comes to forward auth. Any help is appreciated.

**Reply by yamikeigo** [reply]
I salute your efforts to get the ball rolling on this
